src/NUnitFramework/tests/Assertions/LowTrustFixture.cs

```diff
@@ -61,7 +61,12 @@ public void AssertEqualityInLowTrustSandBox()
             });
         }
 
-        [Test, Platform(Exclude="Mono,MonoTouch", Reason= "Mono does not implement Code Access Security")]
+        //[Test, Platform(Exclude="Mono,MonoTouch", Reason= "Mono does not implement Code Access Security")]
+        // TODO: This test fails to run correctly when Assert.Throws
+        // attempts to create an isolated TestExecutionContext.
+        // The TestExecutionContext class would have to be modified
+        // in order to allow chaining of contexts across a 
+        // remoting boundary.
```

Comment:
I thought of a way to fix it so I will create an issue. We used to have a request for attributing tests to run in a separate domain, but I think we got rid of it. We also have a few issues around user-created AppDomains ( #42 and #71 ) which may have a similar cause to this. I'll put details in the issue.